### PR TITLE
Venue Details on List View

### PR DESCRIPTION
**Pull request by EugeneKyale**
Ticket: [TEC-4731](https://theeventscalendar.atlassian.net/browse/TEC-4731)

Ensures the venue country is displayed in the list view when venues are created through the block editor interface.

**Screenshot 📸**

image

[TEC-4731]: https://theeventscalendar.atlassian.net/browse/TEC-4731?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by redscar — DISMISSED** [review]

**Review by redscar — DISMISSED** [review]

File: readme.txt [hunk 1]
```diff
@@ -231,9 +231,10 @@ Remember to always make a backup of your database and files before updating!
 
 = [TBD] TBD =
 
+* Fix - Ensure the venue country is displayed in the list view when venues are created through the block editor interface. [TEC-4731]
+* Fix - Make improvements to the canonical tags added to event views to improve compatibility with SEO plugins. [TEC-4693]
 * Tweak - Added Filters: `tec_events_linked_posts_my_posts_post_status`, `tec_events_linked_posts_all_posts_post_status` to allow filtering of post status of Linked Posts. [CE-27]
 * Tweak - Replace the use of `FILTER_SANITIZE_STRING` in favour of `tec_sanitize_string` to improve PHP 8.1 compatibility. [TEC-4666]
-* Fix - Make improvements to the canonical tags added to event views to improve compatibility with SEO plugins. [TEC-4693]
```
Comment (Camwyn) [inline readme.txt:236]: Be  careful about this - in another PR you moved it to the top. We can move them during the deploy process (and avoid conflicts) 😉

File: tests/views_integration/Tribe/Events/Views/V2/Partials/List_View/Event/__snapshots__/VenueTest__test_render_with_event_with_venue__1.php [hunk 2]
```diff
@@ -2,6 +2,6 @@
 	<span class="tribe-events-calendar-list__event-venue-title tribe-common-b2--bold">
 		Venue 2233 title	</span>
 	<span class="tribe-events-calendar-list__event-venue-address">
-		100 Rue de Le Chat, Paris, Ile de France	</span>
```
Comment (Camwyn) [inline tests/views_integration/Tribe/Events/Views/V2/Partials/List_View/Event/__snapshots__/VenueTest__test_render_with_event_with_venue__1.php:5]: Can we add a second test that includes a state/province as well? To be sure it doesn't get duplicated accidentally.

**Review by Camwyn — CHANGES_REQUESTED** [review]
Just needs one more test.

File: tests/views_integration/Tribe/Events/Views/V2/Partials/List_View/Event/__snapshots__/VenueTest__test_render_with_event_with_venue__1.php (hunk 2 above)
Comment (Camwyn) [reply tests/views_integration/Tribe/Events/Views/V2/Partials/List_View/Event/__snapshots__/VenueTest__test_render_with_event_with_venue__1.php:5]: :doh:

**Review by Camwyn — APPROVED** [review]

**Pull request merged**